packages/store-sync/src/createStoreSync.ts

```diff
@@ -199,7 +201,28 @@ export async function createStoreSync({
   let endBlock: bigint | null = null;
   let lastBlockNumberProcessed: bigint | null = null;
 
-  const storedBlock$ = combineLatest([startBlock$, latestBlockNumber$]).pipe(
+  const storedIndexerLogs$ = indexerUrl
+    ? startBlock$.pipe(
+        mergeMap((startBlock) => {
+          const url = new URL(
+            `api/logs-live?${new URLSearchParams({
+              input: JSON.stringify({ chainId, address, filters }),
+              block_num: startBlock.toString(),
+              include_tx_hash: "true",
+            })}`,
+            indexerUrl,
+          );
+          return fromEventSource<string>(url);
+        }),
+        map((messageEvent) => JSON.parse(messageEvent.data) as StorageAdapterBlock),
```

Comment:
is this a whole block? not a log?

Reply:
It's a whole block, but it's true that the type is not quite accurate, it comes back as a string from the indexer (same for the `blockNumber` on each log). Thinking about the best way to handle this, would like to avoid having to iterate through every log here to turn the string into a bigint :/

Reply:
never mind, the block number only appears once in the response, so we don't have to iterate through the logs!

Reply:
if this came back as an RPC log (i.e. hex encoding), we could use viem to convert things like bigints etc.

if this is a "sql api" response, could we use our tools to convert between sql api responses and JS types?

Reply:
It's the same response that we get from the `/logs` api, reused the same helper to verify the data matches the expected format now